plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/general/SwitchGroup/index.tsx

```diff
@@ -0,0 +1,151 @@
+import AddButton from "@web/extensions/sidebar/core/components/content/common/AddButton";
+import { Icon, Dropdown, Menu } from "@web/sharedComponents";
+import { styled } from "@web/theme";
+
+import { BaseFieldProps } from "../../types";
+
+import useHooks from "./hooks";
+import SwitchGroupItems from "./SwitchGroupItem";
+
+type Props = BaseFieldProps<"switchGroup">;
+
+const SwitchGroup: React.FC<Props> = ({ value, editMode }) => {
```

Comment:
Just use BaseFieldProps<"switchGroup"> directly instead of making the Props type.